app/models/participant_declaration/npq.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class ParticipantDeclaration::NPQ < ParticipantDeclaration
+  belongs_to :npq_lead_provider, through: :cpd_lead_provider
```

Comment:
This probably should be 'has_one` - `belongs_to` in rails land means the id of the owner lives on this object.

So, `school_cohort` `belongs_to` `school` means the school cohort has school id on it. And in this sense, `through` can't work.

I reckon that's why the app is not starting.

Reply:
Oops. Quite right.

Reply:
Wasn't actually adding NPQ into this PR. That shouldn't be there.